.github/pull_request_template.md

```diff
@@ -0,0 +1,10 @@
+<!--- Provide a general summary of your changes in the Title above -->
+<!--- Consider using Conventional Commits prefixes like feat, fix, docs -->
+<!-- https://www.conventionalcommits.org/en/v1.0.0/ -->
+
+## Description
+<!--- Describe your changes in detail -->
+
+## Changelog?
+ - [ ] YES: This PR warrants a changelog entry and I have added one.
+ - [ ] NO: This PR does not warrant a changelog entry
```

Comment:
could see consolidating this into one check box - "Changelog has been updated as necessary" and changing the header from `Changelog?` to something that will be generic enough to encompass future check boxes (e.g. "has a security review been completed") like `Assertions` or `Contributor's Checklist`

Reply:
agreed, updated